### PR TITLE
normalize tf frontend

### LINKED ISSUE #7565: normalize

### PR DESCRIPTION
**Pull request by sondertg**
close #7565

### CODE REVIEW

**Comment by sondertg** [comment]
close #7565

**Comment by sondertg** [comment]
https://github.com/unifyai/ivy/pull/7566#issuecomment-1336335043

File: ivy/functional/frontends/tensorflow/linalg.py [hunk 1]
```diff
@@ -96,19 +97,14 @@ def norm(tensor, ord="euclidean", axis=None, keepdims=None, name=None):
 
 
 @to_ivy_arrays_and_back
+@with_supported_dtypes({"2.9.0 and below": ("float32", "float64")}, "tensorflow")
 def normalize(tensor, ord="euclidean", axis=None, name=None):
+    tensor = tf_frontend.convert_to_tensor(tensor, dtype=ivy.dtype(tensor), dtype_hint="Any")
```
Comment (Daniel4078) [inline ivy/functional/frontends/tensorflow/linalg.py:102]: ivy/functional/frontends/tensorflow/linalg.py:102:89: E501 line too long (93 > 88 characters)

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
Just this one small formatting error for you to change. Thank you.

**Review by Daniel4078 — APPROVED** [review]
LGTM

**Pull request merged**